onnxruntime/core/providers/dml/DmlExecutionProvider/src/ExecutionProvider.cpp

```diff
@@ -624,18 +624,20 @@ namespace Dml
 
     bool IsCpuOnDmlOperator(const onnxruntime::Node& node)
     {
-        auto sequence_ops = std::array<char*, 6>{
+        auto cpu_on_dml_ops = std::array<char*, 8>{
```

Comment:
```suggestion
        auto cpuOnDmlOperators = std::array<char*, 8>{
```
Intrafile consistency.